lib/util/array.ts

```diff
@@ -0,0 +1,8 @@
+import is from '@sindresorhus/is';
+
+export function coerceArray(input: unknown): any[] {
```

Comment:
Not sure of types here. The entire problem here is that we define Types of what we expect the file to be, without validating that it complies with those Types. `unknown[]` would be the honest response but requires more changes